app/javascript/packs/application.tsx

```diff
@@ -147,7 +147,9 @@ initReact({
       trackTitle={data.track_title}
       exerciseTitle={data.exercise_title}
       introduction={data.introduction}
-      instructions={data.instructions}
+      instructions={
+        (camelizeKeys(data.instructions) as unknown) as ExerciseInstructions
```

Comment:
This was required to have the nested `generalHints` property be populated from the `general_hints` property